Zend/tests/operators/_helper.inc

```diff
@@ -0,0 +1,114 @@
+<?php
+
+function get_test_values() {
```

Comment:
Do we really need all these values? For example 
```php
['foo' => 1, 'bar' => 2],
['bar' => 1, 'foo' => 2],
```

aren't they the same?

Reply:
Not every combination is needed, but all values serve a purpose as operand of one of the relational operators. The point of these tests is to ensure that changes do not change the current behavior of the operators.

In your example; An array in PHP is actually an ordered map. The ordering matters, especially for comparison operators.

```php
$a = ['foo' => 1, 'bar' => 2];
$b = ['bar' => 1, 'foo' => 2];

$a < $b; // true
$b < $a; // true
```

That's because the items are compared in order. So in the first statement the operator will first compare `$a['foo'] < $b['foo']`. In the second statement, the operator does `$b['bar'] < $a['bar']`. Both evaluate to true, so both statements are true.